LEVEL2/2884.py

```diff
@@ -0,0 +1,14 @@
+h, m = map(int, input().split())
+
+if m >= 45:
+    print(h, m - 45)
+elif h > 0:
+    print(h - 1, 15 + m) # 45분보다 작으면 시에서 1을 빼야함
+else:
+    print(23, 15 + m) # 0시일 때 가정. 45분보다 작아 시에서 1을 빼야하므로 23시라 두었음.
```

Comment:
고민했던 흔적이 너무 좋네욯ㅎ
if 문 하나 더 줄일 수 있어요~~! :grin: 
```suggestion
else:
    print((h-1) % 24, 15 + m) # 45분보다 작으면 시에서 1을 빼야함
```